### PR TITLE
m-ruslan/movie-seat-booking is ready for review

### PR DESCRIPTION
**Pull request by m-ruslan**
### Movie Seat Booking

[Demo](https://m-ruslan.github.io/html-movie-seat-booking/)
[Code base](https://github.com/m-ruslan/html-movie-seat-booking)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/m-ruslan/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="./img/cinema.png" />
+    <title>Document</title>
```
Comment (A-Ostrovnyy) [inline submissions/m-ruslan/html-movie-seat-booking/index.html]: Add an appropriate title.

File: submissions/m-ruslan/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+</head>
+
+<body>
+
+    <header>
+        <img class="header__logoImg" src="./img/cinema.png" alt="Best Cinema Logo">
```
Comment (A-Ostrovnyy) [inline submissions/m-ruslan/html-movie-seat-booking/index.html]: The logo must be a link to the home page.

File: submissions/m-ruslan/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    </main>
+    <footer>
+        <h2 class="footer__header">Best Cinema LLC</h2>
+        <ul class="footer__list">
+
+            <li>
+                <address>13 Center Street, Capital City</address>
+            </li>
+            <li>tel: <a href="tel:+980991516171">+980991516171</a></li>
+            <li>email: <a href="mailto:bestcinema@gmail.com">bestcinema@gmail.com</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/m-ruslan/html-movie-seat-booking/index.html:175]: 🔥

File: submissions/m-ruslan/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        </ul>
+    </footer>
+</body>
+
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/m-ruslan/html-movie-seat-booking/index.html]: Add an empty line at the end of each file.

File: submissions/m-ruslan/html-movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+<body>
+
+    <header>
+        <img class="header__logoImg" src="./img/cinema.png" alt="Best Cinema Logo">
+        <h2 class="header__header">Best Cinema</h2>
+
+    </header>
+    <main>
+        <section class="movie">
+
```
Comment (A-Ostrovnyy) [inline submissions/m-ruslan/html-movie-seat-booking/index.html]: Reduce unused lines.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Remove files, not for review(img etc.)
See my comments below!

**Comment by m-ruslan** [comment]
@A-Ostrovnyy thank you for the review. I've made fixes in code, please re-review)

File: submissions/m-ruslan/html-movie-seat-booking/index.html (hunk 5 above)
Comment (m-ruslan) [reply submissions/m-ruslan/html-movie-seat-booking/index.html]: Fixed)

File: submissions/m-ruslan/html-movie-seat-booking/index.html (hunk 4 above)
Comment (m-ruslan) [reply submissions/m-ruslan/html-movie-seat-booking/index.html]: Fixed)

File: submissions/m-ruslan/html-movie-seat-booking/index.html (hunk 2 above)
Comment (m-ruslan) [reply submissions/m-ruslan/html-movie-seat-booking/index.html]: Thank you. Due to we have only one page I've added the blank link.

File: submissions/m-ruslan/html-movie-seat-booking/index.html (hunk 1 above)
Comment (m-ruslan) [reply submissions/m-ruslan/html-movie-seat-booking/index.html]: Fixed)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by m-ruslan** [comment]
@A-Ostrovnyy thank you!)

**Pull request merged**